lib/transaction/output.js

```diff
@@ -1,17 +1,16 @@
 'use strict';
 
-var _ = require('lodash');
-var BN = require('../crypto/bn');
-var buffer = require('buffer');
-var bufferUtil = require('../util/buffer');
-var JSUtil = require('../util/js');
-var BufferWriter = require('../encoding/bufferwriter');
-var Script = require('../script');
-var $ = require('../util/preconditions');
-var errors = require('../errors');
-var Opcode = require('../opcode');
-var BufferUtil = require('../util/buffer');
-var MAX_SAFE_INTEGER = 0x1fffffffffffff;
+const _ = require('lodash');
+const BN = require('../crypto/bn');
+const buffer = require('buffer');
+const bufferUtil = require('../util/buffer');
+const JSUtil = require('../util/js');
+const BufferWriter = require('../encoding/bufferwriter');
+const Script = require('../script');
+const $ = require('../util/preconditions');
+const errors = require('../errors');
+const Opcode = require('../opcode');
+const MAX_SAFE_INTEGER = 0x1fffffffffffff;
 
 function Output(args) {
```

Comment:
Do we want to add doc description for this class?

Reply:
Same